ApartmentServicePortal/src/main/java/com/asp/society/models/ERole.java

```diff
@@ -0,0 +1,6 @@
+package com.asp.society.models;
+
+public enum ERole {
+
+    ROLE_USER, ROLE_MODERATOR, ROLE_ADMIN
```

Comment:
Try to give short names for enum like `ROLE_USER('U')` . It will be useful to save space in DB

Reply:
Will modify it in next PR.